docs/guides/base-images.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: How to use parent images
+description: "A guide to configuring a parent image for your truss"
+---
+
+
+Bring your own parent image with Truss.
```

Comment:
I think we should continue to use `base_image` terminology instead of parent. It's the name of the config option, and feels a little clearer